Formula/monero.rb

```diff
@@ -27,6 +27,13 @@ class Monero < Formula
     sha256 "27d1f56406ba94ee779e639203218820975cf68174f92fbeae0f645df0fcada4"
   end
 
+  # Upstream commit for Boost 1.70.0
+  # https://github.com/monero-project/monero/pull/5328
+  patch do
+    url "https://github.com/monero-project/monero/commit/17769db9.diff?full_index=1"
```

Comment:
this patch doesn't apply

Reply:
it probably needs this monstrosity of over 4k lines to apply: https://github.com/monero-project/monero/commit/973403bc9f54ab0722b67a3c76ab6e7bafbfeedc